### ISSUE TITLE
SC 2.4.11 "Focus Appearance" on WCAG 2.2 Candidate Recommendation

### ISSUE THREAD

**Issue by caztcha** [w3c/wcag#2689]
As for the SC 2.4.11 "Focus Appearance" (currently "at risk" status), I feel that it is too complex and difficult to decipher in practical content production. I understand that each SC needs to be "testable" and I appreciate AGWG has done very tough challenges to make the "Focus Appearance” criterion to be testable.

I myself think that this SC 2.4.11 should **not** be deleted, because the focus appearance with sufficient color contrast will surely help many web users. How about considering to make this SC simpler? I think the essential points are the followings.

- Each focus indicator has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused states.
- Each focus indicator has a contrast ratio of at least 3:1 against adjacent non-focus-indicator colors.

That's it. (Or, delete the list item "2" in the current SC 2.4.11.)

If this simpler SC is too hard for many websites which already conform to the level AA, it might be worth considering to change this SC to AAA. 

On the other hand, I think it is smarter to solve this problem on the user agent (or browser) side, not the content author side. How about including this kind of criterion in UAAG?

**Comment by alastc** [comment]
Suggested response for review by the working group:

<hr>

The omission of an area requirement means there is very little increase in requirement compared to focus visible + non-text contrast. ([That section of understanding document](url).)

A 1 pixel square with contrast would pass. 

Although it adds complexity, a size requirement is a key part of focus-appearance.

Perhaps unintuitive, but the simpler version of the SC (the first part) is harder to meet, or at least is much more restrictive in which indicators would work, creating false-positives as it would fail indicators with good visibility.

As proposed, the version above would fail large indicators with gradients / shadows, and ones which have small overlaps with other components (impacting the contrast).
